### ISSUE TITLE
Unit tests needed for Open MCT - YAMCS adapter

### ISSUE THREAD

**Issue by akhenry** [akhenry/openmct-yamcs#14]
If contributing unit tests, take a look at our [testing standards](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md#test-standards).

**Comment by unlikelyzero** [comment]
### Testing Notes
Run `npm test` and get a non-0 exit code

**Comment by khalidadil** [comment]
I'm getting the following:
```
> openmct-yamcs@2.0.5 test /Users/mkadil/Documents/openmct-yamcs
> echo "Error: no test specified" && exit 1

Error: no test specified
npm ERR! Test failed.  See above for more details.
```

Is that expected @unlikelyzero?

**Comment by khalidadil** [comment]
I'm seeing errors when testing this, reopening

**Comment by scottbell** [comment]
@khalidadil @unlikelyzero This works for me now without issue:
```
$ npm test

> openmct-yamcs@2.2.4-SNAPSHOT test
> npm run test:getopensource && npm run test:e2e:smoke


> openmct-yamcs@2.2.4-SNAPSHOT test:getopensource
> sh ./tests/git-opensource-tests.sh

Cloning into 'e2e/opensource'...
remote: Enumerating objects: 1294, done.
remote: Counting objects: 100% (1294/1294), done.
remote: Compressing objects: 100% (958/958), done.
remote: Total 1294 (delta 387), reused 972 (delta 305), pack-reused 0
Receiving objects: 100% (1294/1294), 1.60 MiB | 4.86 MiB/s, done.
Resolving deltas: 100% (387/387), done.

> openmct-yamcs@2.2.4-SNAPSHOT test:e2e:smoke
> npx playwright test --config=./tests/e2e/playwright-quickstart.config.js --project=chromium quickstartSmoke

 DeploySentinel Reporter: [v0.3.4] is disabled (key not found) 

Running 2 tests using 1 worker

[WebServer] <i> [webpack-dev-server] [HPM] Proxy created: /yamcs-proxy  -> http://0.0.0.0:8090/
[WebServer] <i> [webpack-dev-server] [HPM] Proxy rewrite rule created: "^/yamcs-proxy/" ~> ""
[WebServer] <i> [webpack-dev-server] [HPM] Proxy created: /yamcs-proxy-ws  -> ws://0.0.0.0:8090/api/websocket
[WebServer] <i> [webpack-dev-server] [HPM] Proxy rewrite rule created: "^/yamcs-proxy-ws/" ~> ""
[WebServer] <i> [webpack-dev-server] Project is running at:
[WebServer] <i> [webpack-dev-server] Loopback: http://localhost:9000/
<i> [webpack-dev-server] On Your Network (IPv4): http://192.168.22.10:9000/
<i> [webpack-dev-server] On Your Network (IPv6): http://[fe80::1]:9000/
<i> [webpack-dev-server] Content not from webpack is served from '/Users/scott/workspace/openmct-yamcs/example, /Users/scott/workspace/openmct-yamcs/node_modules/openmct/dist' directory
[WebServer] <i> [webpack-dev-middleware] wait until bundle finished: /
[WebServer] <i> [webpack-dev-server] [HPM] Upgrading to WebSocket
  ✓  1 …amcs › Verify that the create button appears and that the Folder Domain Object is available for selection (2.9s)
[WebServer] <i> [webpack-dev-server] [HPM] Client disconnected
[WebServer] <i> [webpack-dev-server] [HPM] Upgrading to WebSocket
[WebServer] <i> [webpack-dev-server] [HPM] Upgrading to WebSocket
[WebServer] <i> [webpack-dev-server] [HPM] Client disconnected
  ✓  2 …e2e.spec.js:50:5 › Quickstart smoke tests @yamcs › Verify that the default yamcs items appear in the tree (1.8s)
[WebServer] <i> [webpack-dev-server] [HPM] Upgrading to WebSocket
[WebServer] <i> [webpack-dev-server] [HPM] Client disconnected

  2 passed (6.0s)
::notice title=🎭 Playwright Run Summary::  2 passed (6.1s)
```

**Comment by scottbell** [comment]
Closing for the time being.
